### ISSUE TITLE
[Master] Name, Role, Value

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#11694]
## What is Name, Role, Value
In a nutshell: Providing role, state, and value information on all user interface components enables compatibility with assistive technology (such as screen readers, screen magnifiers, and speech recognition software).

E.g. Using aria-label to provide an invisible label where a visible label cannot be used. Using ARIA state and property attributes to expose the state of a user interface component. Using ARIA role to expose the role of a user interface component.

## Overview
- Success Criterion: WCAG 2.1 - 4.1.2
- **MASTER** Ticket #11672
- [**Github label:**](https://github.com/SAP/spartacus/issues?q=is%3Aissue+is%3Aopen+label%3Aa11y-sr%2Fname-role-value) `a11y-sr/name-role-value`

## Implementation

- [x] #11637
- [x] #13572
- [x] #13432 
- [x] #12898
- [x] #3608
- [x] #13432
- [x] #13511
- [x] #13516
- [x] #13517 
- [x] #13518 
- [x] #13519 
- [x] #13520
- [x] #13521
- [x] #13522
- [x] #13561 
- [x] #13717
- [x] #13718
- [x] #13724

**Comment by developpeurweb** [comment]
all tickets done
